### DISCUSSION TITLE
Signup using group email issue

### DISCUSSION THREAD

**Discussion by IbrahimKhanLS** [codemagic-ci-cd/codemagic-docs#2651] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Sign up using a group email doesn't send verification code to the email.

### Expected results

Expecting to receive a verification code on group email

### Actual results

Not receiving a verification code on group email

**Comment by dtrdic** [comment]
Hello @IbrahimKhanLS is the email containing the authentication key not arriving at all, or is there a delay in its delivery? Please check your spam/junk folders as well. If there is a delay, it might be caused by spam/virus filters on your end. 
If you have implemented any spam filters, consider whitelisting our domain [no-reply@codemagic.io](mailto:no-reply@codemagic.io)

**Reply by IbrahimKhanLS** [reply]
@dtrdic I am not receiving key at all, neither in spam/junk.

**Reply by dtrdic** [reply]
@IbrahimKhanLS can you please share with me the email you are using to log in? I'll remove this email from the thread as soon as I check the logs

**Reply by dtrdic** [reply]
@IbrahimKhanLS thanks for sharing. Please try now

**Reply by IbrahimKhanLS** [reply]
@dtrdic Thanks, highly appreciated.
